classes/ActionScheduler_StoredAction.php

```diff
@@ -5,22 +5,26 @@
  *
  * An action which has been saved in the data store and is pending execution (i.e. not finished).
  */
-class ActionScheduler_StoredAction extends ActionScheduler_Action {
+class ActionScheduler_StoredAction extends ActionScheduler_AbstractAction implements ActionScheduler_Interface_Storable, ActionScheduler_Interface_Claimable {
```

Comment:
This should be `ActionScheduler_Stored_Action` for consistency. Assuming it's possible rename the class at this point? If not, we can leave it for another day.